stdlib/2and3/mailbox.pyi

```diff
@@ -0,0 +1,194 @@
+
+    def __init__(self, dirname: _PathType, factory: Optional[Callable[[IO[Any]], mboxMessage]] = ..., create: bool = ...) -> None: ...
+
+class MMDF(_mboxMMDF[MMDFMessage]):
+
+    def __init__(self, dirname: _PathType, factory: Optional[Callable[[IO[Any]], MMDFMessage]] = ..., create: bool = ...) -> None: ...
+
+class MH(Mailbox[MHMessage]):
+
+    def __init__(self, dirname: _PathType, factory: Optional[Callable[[IO[Any]], MHMessage]] = ..., create: bool = ...) -> None: ...
+    def get_file(self, key: str) -> _ProxyFile[bytes]: ...
+
+    def list_folders(self) -> List[str]: ...
+    def get_folder(self, folder: _PathType) -> MH: ...
+    def add_folder(self, folder: _PathType) -> MH: ...
+    def remove_folder(self, folder: _PathType) -> None: ...
+    def get_sequences(self) -> Dict[str, List[int]]: ...
+    def set_sequences(self, sequences: Dict[str, List[int]]) -> None: ...
+    def pack(self) -> None: ...
+
+class Babyl(_singlefileMailbox[BabylMessage]):
+
+    def __init__(self, dirname: _PathType, factory: Optional[Callable[[IO[Any]], BabylMessage]] = ..., create: bool = ...) -> None: ...
+
+    def get_labels(self) -> List[str]: ...
+
+class Message(email.message.Message):
+
+    def __init__(self, message: Optional[_MessageData] = ...) -> None: ...
+
+class MaildirMessage(Message):
+
+    def get_subdir(self) -> str: ...
+    def set_subdir(self, subdir: Union[Literal["new"], Literal["cur"]]) -> None: ...
+    def get_flags(self) -> str: ...
+    def set_flags(self, flags: Iterable[str]) -> None: ...
+    def add_flag(self, flag: str) -> None: ...
+    def remove_flag(self, flag: str) -> None: ...
+    def get_date(self) -> int: ...
+    def set_date(self, date: int) -> None: ...
+    def get_info(self) -> str: ...
+    def set_info(self, info: str) -> None: ...
+
+class _mboxMMDFMessage(Message):
+
+    def get_from(self) -> str: ...
+    def set_from(self, from_: str, time_: Optional[Union[bool, Tuple[int, int, int, int, int, int, int, int, int]]] = ...) -> None: ...
+    def get_flags(self) -> str: ...
+    def set_flags(self, flags: Iterable[str]) -> None: ...
+    def add_flag(self, flag: str) -> None: ...
+    def remove_flag(self, flag: str) -> None: ...
+
+class mboxMessage(_mboxMMDFMessage): ...
+
+class MHMessage(Message):
+
+    def get_sequences(self) -> List[str]: ...
+    def set_sequences(self, sequences: Iterable[str]) -> None: ...
+    def add_sequence(self, sequence: str) -> None: ...
+    def remove_sequence(self, sequence: str) -> None: ...
+
+class BabylMessage(Message):
+
+    def get_labels(self) -> List[str]: ...
+    def set_labels(self, labels: Iterable[str]) -> None: ...
+    def add_label(self, label: str) -> None: ...
+    def remove_label(self, label: str) -> None: ...
+    def get_visible(self) -> Message: ...
+    def set_visible(self, visible: _MessageData) -> None: ...
+    def update_visible(self) -> None: ...
+
+    def get_file(self, key: str) -> IO[bytes]: ...
```

Comment:
I don't see this method in 3.8.

Reply:
This was meant to be on Babyl, not BabylMessage. My mistake